src/sentry/search/snuba/backend.py

```diff
@@ -36,26 +36,39 @@ def assigned_to_filter(actor, projects):
                 team=actor, project_id__in=[p.id for p in projects]
             ).values_list("group_id", flat=True)
         )
+    elif isinstance(actor, User) or (isinstance(actor, list) and actor[0] == "me_or_none"):
```

Comment:
Do we actually need this elif?

Reply:
I felt safer having it in, together with `raise InvalidSearchQuery("Unsupported assignee type.")`...but given we've gone so long without this, I just removed it all. Wonder if I should do the same for `assigned_or_suggested` now..